docs/editor_integration.md

```diff
@@ -1,14 +1,23 @@
 For a smother experience, you can run `autoimport` each time you save your file
-in your editor.
+in your editor or make a commit.
 
-# Vim
+# [pre-commit](https://pre-commit.com/)
 
-To integrate `autoimport` into Vim, I recommend using the [ale
-plugin](https://github.com/dense-analysis/ale).
+You can run `autoimport` before we do a commit using the
+[pre-commit](https://pre-commit.com/) framework. If you don't know how to use
+it, follow [these
+guidelines](https://lyz-code.github.io/blue-book/devops/ci/#configuring-pre-commit).
 
-!!! note ""
+You'll need to add the following lines to your project's
+`.pre-commit-config.yaml` file.
 
-    If you are new to ALE, check [this
-    post](https://lyz-code.github.io/blue-book/linux/vim/vim_plugins/#ale).
+```yaml
+repos:
+  - repo: https://github.com/lyz-code/autoimport/
+    rev: master
+    hooks:
+      - id: autoimport
+```
 
-`ale` is configured to run `autoimport` automatically by default.
+# Vim
```

Comment:
Instead of removing the `vim` section, I meant to preserve the existent content.

Can you please restore it?

Thanks!

Reply:
Restored. My apologies!